modules/data-loading/pages/kafka-loader-user-guide.adoc

```diff
@@ -127,10 +257,10 @@ GSQL > CREATE DATA_SOURCE KAFKA k1 = "/path/to/config" FOR GRAPH test_graph
 
 [NOTE]
 ====
-In the above statement, the local data_source k1 is only accessible to graph test_graph. A superuser cannot grant it to another graph**.**
+In the above statement, the local data_source k1 is only accessible to graph test_graph. A superuser cannot grant it to another graph.
 ====
 
-==== DROP DATA_SOURCE
+=== DROP DATA_SOURCE
 
 A data source variable can be dropped by a user who has privilege. A global data source can only be dropped by a users with global `WRITE_DATASOURCE` privilege. Users with `WRITE_DATASOURCE` privilege for one graph can drop data sources on that graph. The syntax for the `DROP DATA_SOURCE` command is as follows:
```

Comment:
A data source variable can be dropped by a user who has the appropriate level of privileges. A global data source can only be dropped by users with global `WRITE_DATASOURCE` privileges. Users with `WRITE_DATASOURCE` privileges for one graph can drop data sources on that graph. The syntax for the `DROP DATA_SOURCE` command is as follows: